firefox-ios/Client/Frontend/TrackingProtection/TrackingProtectionDetailsViewController.swift

```diff
@@ -107,13 +114,13 @@ class TrackingProtectionDetailsViewController: UIViewController, Themeable {
             scrollView.topAnchor.constraint(equalTo: headerView.bottomAnchor),
             scrollView.bottomAnchor.constraint(
                 equalTo: view.bottomAnchor,
-                constant: -TPMenuUX.UX.TrackingDetails.bottomDistance
+                constant: -UX.bottomDistance
             ),
             scrollView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
             scrollView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
             baseView.topAnchor.constraint(
                 equalTo: scrollView.topAnchor,
-                constant: TPMenuUX.UX.TrackingDetails.baseDistance
+                constant: UX.baseDistance
```

Comment:
Yay! 🎉  Just what I like to see!

Reply:
For sure, where was possible, I did that. :)